site/docs/operations/configuration.md

```diff
@@ -138,6 +138,7 @@ The most important properties are listed here:
 |------|:------|----------|
 |hazelcast.partition.count|271|Total number of partitions in the cluster.|
 |hazelcast.logging.type|jdk|What logger should be used by Jet. Valid options are `log4j`, `log4j2`, `slf4j` and `none`|
+|hazelcast.lite-member.enabled|false|Specifies whether the IMDG instance behind the Jet is data member or lite member. In our case where IMDG is used as Jet data storage, we recommend that you do not change the default value, as it increases the latency, the default value is false, i.e., data member. The reason for this latency increase is that accessing data in IMDG partitions has to contain network calls in the lite member. Besides, Jet needs at least one data member in the cluster to run jobs because job coordination primitives are kept in IMDG data structures.|
```

Comment:
"whether the Jet instance acts as a data member or a lite member at the IMDG level. Since Jet uses IMDG services for data storage, we..."

Reply:
I couldn't find this setting in `JetProperties` or `ClusterProperty`...

Reply:
I realized it's in the wrong place. But, where should I move this? (to the upper section?)

Reply:
I removed it.